src/jetstream/repository/interfaces/tokens.go

```diff
@@ -10,6 +10,7 @@ type Token struct {
 // TokenRepository is an application of the repository pattern for storing tokens
 type TokenRepository interface {
 	FindAuthToken(userGUID string, encryptionKey []byte) (TokenRecord, error)
+	ListAuthToken(encryptionKey []byte) ([]TokenRecord, error)
```

Comment:
This was used by a previous implementation, but i forgot to also remove the function in the TokenRepository. Since it's not used anymore, i will remove it.